templates/endpoints/models/meta-llama--Llama-2-13b-chat-hf.yaml

```diff
@@ -0,0 +1,47 @@
+# This model requires HUGGING_FACE_HUB_TOKEN environment variable to be set to a Hugging Face
+# Access Token for an account with permissions to download the model.
+deployment_config:
+  autoscaling_config:
+    min_replicas: 1
+    initial_replicas: 1
+    max_replicas: 1
+    target_num_ongoing_requests_per_replica: 400.0
```

Comment:
this should be around 60. 400 seems a lot :D

Lets also limit max_concurrent_queries to 80 or 90